test/Makefile

```diff
@@ -1,6 +0,0 @@
-all: bins
```

Comment:
i know that some people does use this makefile to fetch the testbins.

Reply:
yeah true! me too

Reply:
Well, they shouldn't.